apiserver/logsink/logsink.go

```diff
@@ -253,6 +348,8 @@ func (h *logSinkHandler) receiveLogs(socket *websocket.Conn, endpointVersion int
 				// The ServeHTTP handler has stopped.
 				return
 			case logCh <- m:
+				h.metrics.LogReadCount(resolvedModelUUID, metricLogReadLabelSuccess).Inc()
```

Comment:
this number will equal the writes, both success and failure, so I'm not sure we need this one.

Reply:
Yes, this would actually. OK, let me rethink this - will probably end removing.